docs/workflows/standalone/dorado_basecalling.md

```diff
@@ -0,0 +1,252 @@
+| dorado_basecalling | **terra_workspace** | String | The name of your Terra workspace where your samples can be found. For example, if your URL contains `#workspaces/example/my_workspace/` then your workspace name is `my_workspace` |  | Required |
+| create_table_from_array | **cpu** | Int | Number of CPUs to allocate to the task | 1 | Optional |
+| create_table_from_array | **disk_size** | Int | Amount of storage (in GB) to allocate to the task | 25 | Optional |
+| create_table_from_array | **docker** | Int | The Docker container to use for the task | us-docker.pkg.dev/general-theiagen/theiagen/terra-tools:2023-06-21 | Optional |
+| create_table_from_array | **memory** | Int | Amount of memory/RAM (in GB) to allocate to the task | 2 | Optional |
+| dorado_basecall | **cpu** | Int | Number of CPUs to allocate to the task | 8 | Optional |
+| dorado_basecall | **disk_size** | Int | Amount of storage (in GB) to allocate to the task | 100 | Optional |
+| dorado_basecall | **docker** | String | The Docker container to use for the task | us-docker.pkg.dev/general-theiagen/staphb/dorado:0.9.0-cuda12.2.0 | Optional |
+| dorado_basecall | **memory** | Int | Amount of memory/RAM (in GB) to allocate to the task | 32 | Optional |
+| dorado_basecalling | **custom_primers** | File | A FASTA file containing custom primer sequences for PCR primer trimming during demultiplexing. |  | Optional |
+| dorado_basecalling | **demux_notrim** | Boolean | Set to `true` to disable barcode trimming during demultiplexing. | false | Optional |
+| dorado_basecalling | **dorado_model** | String | The model to use during basecalling ('sup' for super accuracy, 'hac' for high accuracy, or 'fast' for high speed). Users may also specify a full model name (see [above](#model-type-selection) for more details). | "sup" | Optional |
+| dorado_demux | **cpu** | Int | Number of CPUs to allocate to the task | 4 | Optional |
+| dorado_demux | **disk_size** | Int | Amount of storage (in GB) to allocate to the task | 100 | Optional |
+| dorado_demux | **docker** | String | The Docker container to use for the task | us-docker.pkg.dev/general-theiagen/staphb/dorado:0.9.0-cuda12.2.0 | Optional |
+| dorado_demux | **memory** | Int | Amount of memory/RAM (in GB) to allocate to the task | 16 | Optional |
+| dorado_trim | **cpu** | Int | Number of CPUs to allocate to the task | 4 | Optional |
+| dorado_trim | **disk_size** | Int | Amount of storage (in GB) to allocate to the task | 100 | Optional |
+| dorado_trim | **docker** | String | The Docker container to use for the task. This is not the most up-to-date image since there is a bug with this Dorado subcommand in v0.9.0-cuda12.2.0 | us-docker.pkg.dev/general-theiagen/staphb/dorado:0.8.3 | Optional |
```

Comment:
please change `Dorado subcommand in v0.9.0-cuda12.2.0` to `Dorado subcommand in v0.9.0`

The `-cuda12.2.0` is just part of the docker image tag that we added to indicate the version of CUDA included in the docker image. Not specific to Dorado installation per se